src/components/FilterElement/FilterElement.tsx

```diff
@@ -162,15 +169,21 @@ function FilterElement(props: Props) {
                                         placeholder="Search users..."
                                         onChange={selected => updateSelectFilter(selected.map(s => s.name))}
                                     />
-                                    : <Typeahead
-                                        style={{ display: "block" }}
-                                        className="select-filter"
-                                        defaultSelected={[props.defaultValue]}
-                                        onChange={selected => updateSelectFilter(selected)}
-                                        options={props.options?.options}
-                                        labelKey={convertTagToName}
-                                        selectHintOnEnter={true}>
-                                    </Typeahead >
+                                    : hasFlag(props.options.type, FilterType.EQUAL) && hasFlag(props.options.type, FilterType.SIMPLE)
+                                        ? <Form.Control isInvalid={!isValid} className="select-filter" defaultValue={props.defaultValue} value={value} as="select" onChange={updateSelectFilter}>
+                                            {getSelectOptions()}
+                                        </Form.Control>
+                                        : hasFlag(props.options.type, FilterType.EQUAL)
+                                            ? <Typeahead
+                                                id={props.options.name}
+                                                style={{ display: "block" }}
+                                                className="select-filter"
+                                                onChange={selected => updateSelectFilter(selected)}
+                                                options={props.options?.options}
+                                                labelKey={convertTagToName}
+                                                autoselect={false}
```

Comment:
What does this setting do?

Reply:
With autoselect={true} the first option is put into the select field and has to be deleted by the user if he wants to select something else.